README.md

```diff
@@ -1,5 +1,5 @@
 ## About this project:
-* This repository houses most of the website code for the Demiurge DSP server.
+* This repository houses most of the website code for the Demiurge LSB server.
```

Comment:
I appreciate you catching this, but I think I want to just axe-murder all those brandings old and new lets just amend that to `Demiurge server`